.github/workflows/lint-test.yaml

```diff
@@ -27,10 +27,10 @@ jobs:
         run: git fetch --prune --unshallow
 
       - name: Create kind cluster
-        uses: helm/kind-action@v1.0.0-rc.1
+        uses: helm/kind-action@v1.0.0
```

Comment:
Version `v1.0.0-rc.1` of the kind-action uses kind 1.17.0. When I try to launch a cluster with 1.17.0 on my notebook, coredns is going to CrashLoopBackOff. Therefore there is no valid DNS server in k8s which would help envoy to discover the contour service.

Maybe the real problem for failling pipelines was only the change here:
https://github.com/bitnami/charts/pull/3381/commits/8bab0b3db8b4b3285d4a1d0c3496dfc71edd102c

Should I revert to the RC actions?

Reply:
yes, please could you revert to RC versions? We would like to manage this in a different PR, please.

Reply:
Done and the tests still are green :partying_face: